Documentation/Index.rst

```diff
@@ -45,32 +45,3 @@ to extend the tool.
     Developer/Index
     Migration/Index
     KnownProblems/Index
-
```

Comment:
I think this block may need a rebase?

Reply:
What do you mean?

Reply:
It got rebased now, all good now it seems.